### PR TITLE
Astro release notes - August 10

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODA0Njg&invite=true

We already merged user permissions last week ahead of this release (https://github.com/astronomer/docs/pull/977), so there's not any more doc changes to make for this release.

### CODE REVIEW

File: astro/user-permissions.md [hunk 1]
```diff
@@ -38,18 +38,18 @@ A Workspace role grants a user some level of access to a specific Workspace. The
...
-| View Airflow connections and Variables   |                      |                      | ✔️                   |
-| Update user roles and permissions        |                      |                      | ✔️                   |
-| Update Airflow connections and Variables |                      |                      | ✔️                   |
-| Invite users to a Workspace              |                      |                      | ✔️                   |
+| View Workspace users                     | ✔️                   | ✔️                   | ✔️                  |
+| View all Deployments in the Cloud UI     | ✔️                   | ✔️                   | ✔️                  |
+| View DAGs in the Airflow UI              | ✔️                   | ✔️                   | ✔️                  |
+| View Airflow task logs                   | ✔️                   | ✔️                   | ✔️                  |
+| Update Deployment configurations         |                      | ✔️                   | ✔️                  |
+| Manually trigger DAG and task runs       |                      | ✔️                   | ✔️                  |
```
Comment (jwitz) [inline astro/user-permissions.md:46]: @chomipi88 Saw this got merged last week and wasn't sure what the original text here meant, is this an accurate read of what this permission encapsulates?

Comment (kwcanuck) [reply astro/user-permissions.md:46]: @jwitz while waiting for a response from @chomipi88, you can review the comments in [PR 977](https://github.com/astronomer/docs/pull/977) for background information on these changes. Piotr requested changes to your edits prior to the merge.

File: astro/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
 
 **Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))
 
+## August 10, 2022
+
+### Updated Organization and Workspace role permissions
+
+Some user roles have received new and modified permissions:
```
Comment (kwcanuck) [inline astro/release-notes.md:22]:
```suggestion
The following user roles have new and modified permissions:
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role now provides complete access to Workspaces, Deployments, and usage across your Organization.
```
Comment (kwcanuck) [inline astro/release-notes.md:24]:
```suggestion
- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
```

**Review by kwcanuck — APPROVED** [review]

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 4]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
 
 **Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))
 
+## August 10, 2022
+
+### Updated Organization and Workspace role permissions
```
Comment (paolaperaza) [inline astro/release-notes.md:20]:
```suggestion
### Updated user permissions for Organization and Workspace roles
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+Some user roles have received new and modified permissions:
+
+- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role now provides complete access to Workspaces, Deployments, and usage across your Organization.
+- Organization Billing Admins can now view [usage](deployment-metrics.md#astro-usage) for all Workspaces in their Organization regardless of their Workspace permissions.
+- Workspace Editors can now delete any Deployments in their Workspace.
```
Comment (paolaperaza) [inline astro/release-notes.md:26]:
```suggestion
- Workspace Editors can now delete any Deployment in their Workspace.
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+### Automatic role assignment for new users authenticating with an identity provider
+
+If your organization has [implemented an identity provider (IdP)](configure-idp.md), any new user who authenticates to Astro through your IdP is now automatically assigned the Organization Member role. This means that users authenticating through your IdP do not need to be invited by email before joining your Organization.
+
+### Additional improvements
+
+- Added guardrails so that you can have a Workspace role only if you have an Organization role in the Organization where the Workspace is located.
```
Comment (paolaperaza) [inline astro/release-notes.md:34]:
```suggestion
- Added a security measure that ensures Workspace roles can only be assigned to users who have an Organization role in the Organization in which the Workspace is hosted. This ensures that a user who does not belong to your Organization cannot be assigned a Workspace role within it.
```

**Review by paolaperaza — APPROVED** [review]

File: astro/release-notes.md [hunk 7]
```diff
@@ -15,6 +15,24 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+### Updated user permissions for Organization and Workspace roles
+
+The following user roles have new and modified permissions:
+
+- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
+- Organization Billing Admins can now view [usage](deployment-metrics.md#astro-usage) for all Workspaces in their Organization regardless of their Workspace permissions.
+- Workspace Editors can now delete any Deployment in their Workspace.
+
+### Automatic role assignment for new users authenticating with an identity provider
```
Comment (paolaperaza) [inline astro/release-notes.md:28]:
```suggestion
### Automatic access for new users authenticating with an identity provider
```

**Pull request merged**